test/jdk/javax/sound/midi/Devices/OutputBuffer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.sound.midi.*;
+
+/**
+ * @test
+ * @key sound
+ * @bug 8074211 8250667
+ * @summary Test if part of the previous sysex message in the output buffer
+ *          is sent again with the next sysex message
+ * @comment This test does not fail when the bug occurs. It requires manual
+ *          monitoring of the output.
+ * @requires os.family == "windows"
```

Comment:
The test does not use any Windows-specific API, this check can be removed so the code will be covered on all platforms.

Reply:
OK, I removed the check.